components/o-forms/src/scss/_radio-box.scss

```
@@ -120,13 +121,15 @@
 		// `:focus` fallback, however some of our supported browsers
 		// do not understand `@supports: selector()`.
 		@supports selector(:focus-visible) {
-			&:focus + .o-forms-input__label { // stylelint-disable-line selector-no-qualifying-type
+			&:focus + .o-forms-input__label {
+				// stylelint-disable-line selector-no-qualifying-type
```

This formatting will result in lint errors, can it be reverted for this line
```suggestion
			&:focus + .o-forms-input__label { // stylelint-disable-line selector-no-qualifying-type
```